Felienne/hedy
# [UI Improvement] Implement and add page titles to each render_template

**Description**
Currently the handling of page titles is a bit of a mess. With the older .md file structure some titles where retrieved from these files whereas the titles of other pages, such as `my-profile` and `for-teachers` were hard-coded or didn't exist. With this PR we align all page title translations into one .json file, enabling us to easier add, translate and retrieve these titles. A function is added to enable each `render_template` to retrieve the correct page title.

**Fix for**
This PR fixed #1449

**How to test**
Navigate through the website and notice that a correct page title is shown for each page. Currently they are only translated in dutch and English. The automatic fallback language is English. If a key in the dict doesn't exist we automatically return the title of the English homepage ("Hedy - A gradual program language").

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
